packages/salesforcedx-utils/src/cli/cliCommandExecutor.ts

```diff
@@ -19,10 +19,7 @@ export class CliCommandExecutor {
     baseEnvironment: Map<string, string>
   ): SpawnOptions {
     // start with current process environment
-    const env = Object.create(null);
-
-    // inherit current process environment
-    Object.assign(env, process.env);
+    const env = process.env;
```

Comment:
Does this change the functionality here where now env is a ref to process.env so if values of env change we're altering the process.env values?